assets/revisions/rev_u4tep9xyu4hc_update_job_documents.py

```diff
@@ -23,8 +23,14 @@
 
 async def upgrade(ctx: MigrationContext):
     ctx.mongo.jobs.update_many({"key": {"$exists": False}}, {"$set": {"key": None}})
-    ctx.mongo.jobs.update_many({"acquired": None}, {"$set": {"acquired": False}})
-    ctx.mongo.jobs.update_many({"archived": None}, {"$set": {"archived": False}})
+    ctx.mongo.jobs.update_many(
+        {"acquired": {"$exists": False}},
+        {"$set": {"acquired": False}},
+    )
```

Comment:
I think this should be set to `True`.

Any jobs that don't have this field will have been completed in the past, which also means they were acquired.